src/app/Library/CrudPanel/Traits/FieldsProtectedMethods.php

```diff
@@ -7,97 +7,6 @@
 
 trait FieldsProtectedMethods
 {
-    /**
-     * If field has entity we want to get the relation type from it.
-     *
-     * @param  array  $field
-     * @return array
-     */
-    public function makeSureFieldHasRelationType($field)
```

Comment:
Ouch! When did these methods become `public` instead of `protected`? Any idea @pxpm ?

Reply:
Is it too late to make them `private` now?... I really didn't want people to be able to use these themselves... it forces us to keep the same names and returns for ever...

Reply:
In v6 we will do it, probably move them around as we discussed. For now would be BC.

I guess it was my fault from the beginning the ones that are public https://github.com/Laravel-Backpack/CRUD/commit/1db1525f96f1f664a9212e64f2d82401fe5c4aac (6 April 2020)

I don't expect much people to be overriding them .. but yeah, it's a BC and we should treat it like.

Reply:
Ugh you're right, pretty messy, can't figure out what's different and what's not. Can you please move the methods to the same order they were in? That should fix the diff and make it easier to push this through.